lib/ripple/callbacks.rb

```diff
@@ -12,7 +12,7 @@ module Callbacks
     included do
       extend ActiveModel::Callbacks
       define_model_callbacks *(CALLBACK_TYPES - [:validation])
-      define_callbacks :validation, :terminator => proc{|_,result| result == false}, :scope => [:kind, :name]
+      define_callbacks :validation, :terminator => proc{|_,result| result.call == false}, :scope => [:kind, :name]
```

Comment:
Emphasizing really strongly that I couldn't care less, wouldn't it be more Rubyesque to do that as `!result.call`, rather than `result.call == false`? Or are we worried about some other falsy value?

Reply:
Other falsy values would halt the callback chain, in particular nil 😱. This follows the existing conventions in use by callback terminators in Rails core as well:
https://github.com/rails/rails/commit/2386daabe7f8c979b453010dc0de3e1f6bbf859d#diff-174733f2db65ef1bc53e3222d6ac0e61R20
https://github.com/rails/rails/commit/2386daabe7f8c979b453010dc0de3e1f6bbf859d#diff-0cd7b1cc8e754e9ab96c72b45770767dR26